conf/broker.conf

```diff
@@ -704,6 +704,9 @@ managedLedgerMinLedgerRolloverTimeMinutes=10
 # Maximum time before forcing a ledger rollover for a topic
 managedLedgerMaxLedgerRolloverTimeMinutes=240
 
+# Maximum ledger size before triggering a rollover for a topic (MB)
+managedLedgerMaxSizePerLedgerMbytes=2048
```

Comment:
Can you also add this config into `conf/standalone.conf`?

Reply:
Thank you for your timely feedback. I have added the configuration in `conf/standalone.conf`.